pkg/remote/resolution/resolver.go

```diff
@@ -34,7 +36,7 @@ type Resolver struct {
 	requester       remoteresource.Requester
 	owner           kmeta.OwnerRefable
 	resolverName    string
-	params          map[string]string
```

Comment:
Could you add context in the commit/PR message about this change?
It doesn't seem to be strictly required for the `v1beta1` version of remote resolution.
It would have been nice and easier to review to have that as a separate change before the `v1beta1` 🙏